### DISCUSSION TITLE
Textarea

### DISCUSSION THREAD

**Discussion by jrbarnes9** [ONSdigital/design-system#139] [Feedback]
Use this place to discuss the [textarea component](https://ons-design-system.netlify.com/components/textarea) in the [ONS Design System](https://ons-design-system.netlify.com)

**Comment by bameyrick** [comment]
The charcount element cannot be a `div` as the wrapping field element is a `p`

**Reply by jrbarnes9** [reply]
#431 resolved

**Comment by jrbarnes9** [comment]
`rows` param should be changed to a responsive unit of measurement based on the amount of characters that can fit in a text area, so the input can reflect the character limit.  This param needs to be added to the macro options table.

**Reply by jrbarnes9** [reply]
#735 resolved

**Comment by Paul-Joel** [comment]
There is a jinja issue on lines 20, 23 and 31 - params.charCheckLimit.limit is checked without confirming the params.charCheckLimit exists first, so the pattern fails in jinja if no limit is set

**Reply by jrbarnes9** [reply]
#1602 resolved
